### PR TITLE
Bug 1200937 - APN Settings: support reading 'mtu' field from apn database.

### PR DESCRIPTION
**Pull request by jessi3py-zz**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83bf3df927a98ae5bcf1badbbc7e3d805e4d392b

File: apps/system/js/operator_variant_handler.js [hunk 1]
```diff
@@ -270,7 +270,8 @@
           'ril.data.passwd': 'password',
           'ril.data.httpProxyHost': 'proxy',
           'ril.data.httpProxyPort': 'port',
-          'ril.data.authtype': 'authtype'
+          'ril.data.authtype': 'authtype',
+          'ril.data.mtu': 'mtu'
```
Comment (gasolin) [inline apps/system/js/operator_variant_handler.js]: the value is missing in related test https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L100

File: apps/system/js/operator_variant_handler.js [hunk 2]
```diff
@@ -279,7 +280,8 @@
           'ril.supl.passwd': 'password',
           'ril.supl.httpProxyHost': 'proxy',
           'ril.supl.httpProxyPort': 'port',
-          'ril.supl.authtype': 'authtype'
+          'ril.supl.authtype': 'authtype',
+          'ril.supl.mtu': 'mtu'
```
Comment (gasolin) [inline apps/system/js/operator_variant_handler.js]: missing in related test https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L68

File: apps/system/js/operator_variant_handler.js [hunk 3]
```diff
@@ -291,7 +293,8 @@
           'ril.mms.mmsc': 'mmsc',
           'ril.mms.mmsproxy': 'mmsproxy',
           'ril.mms.mmsport': 'mmsport',
-          'ril.mms.authtype': 'authtype'
+          'ril.mms.authtype': 'authtype',
```
Comment (gasolin) [inline apps/system/js/operator_variant_handler.js]: add authtype & mtu in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L87

File: apps/system/js/operator_variant_handler.js [hunk 4]
```diff
@@ -300,7 +303,8 @@
           'ril.dun.passwd': 'password',
           'ril.dun.httpProxyHost': 'proxy',
           'ril.dun.httpProxyPort': 'port',
-          'ril.dun.authtype': 'authtype'
+          'ril.dun.authtype': 'authtype',
+          'ril.dun.mtu': 'mtu'
```
Comment (gasolin) [inline apps/system/js/operator_variant_handler.js]: add test in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L75

File: apps/system/js/operator_variant_handler.js [hunk 5]
```diff
@@ -309,7 +313,8 @@
           'ril.ims.passwd': 'password',
           'ril.ims.httpProxyHost': 'proxy',
           'ril.ims.httpProxyPort': 'port',
-          'ril.ims.authtype': 'authtype'
+          'ril.ims.authtype': 'authtype',
+          'ril.ims.mtu': 'mtu'
```
Comment (gasolin) [inline apps/system/js/operator_variant_handler.js]: add test in https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L82

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1ab8ba48dbfb2eeeddc1bf0e2757f713bc228a19

**Pull request merged**